src/main.rs

```diff
@@ -315,6 +323,18 @@ fn get_data_dir(matches: &ArgMatches) -> String {
     }
 }
 
+/// Gets waiting time when frugalos stops.
+fn get_stop_waiting_time(matches: &ArgMatches) -> Result<Duration> {
+    matches.value_of("STOP_WAITING_TIME_MILLIS").map_or_else(
+        || Ok(Duration::from_millis(10)),
```

Comment:
Longer default value of `stop-waiting-time-millis` takes longer time when frugalos stops, so set it to a small value.